Remotion/ObjectBinding/Web/UI/Controls/Validation/BocListValidationResultDispatchingValidator.cs

```diff
@@ -44,6 +50,19 @@ public void DispatchValidationFailures (IBusinessObjectValidationResult validati
 
       if (dispatchToValidationFailureRepository)
         DispatchToValidationFailureRepository(validationResult, bocListControl);
+
+      var rowObjects = bocListControl.Value?.Cast<IBusinessObject>() ?? Array.Empty<IBusinessObject>();
```

Comment:
Why do you need to do the Cast here? BocList has a property ` IReadOnlyList<IBusinessObject>? Value` that should not need casting?

Reply:
I honestly have no clue why that cast was there or any reason to have it there, don't know how that happened.